File: packages/m05-twin-ops/backend/src/main/java/com/comm/m05/entity/Alert.java
package com.comm.m05.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

@Data
@TableName("m05_alert")
public class Alert {
    @TableId(type = IdType.AUTO)
    private Long id;
    private Long deviceId;
    private String deviceCode;
    private String alertContent;
    private Integer level;
    private Integer status;
    private String source;
    private String orderNo;
    private LocalDateTime createTime;
    private LocalDateTime updateTime;
}

File: packages/s5-construction-monitor/twin-csharp/TwinCsharp/Models/Alert.cs
namespace TwinCsharp.Models;

/// <summary>
/// 告警实体。字段命名/类型对齐 packages/m05-twin-ops/backend 的 Alert.java。
/// </summary>
public class Alert
{
    public long Id { get; set; }
    public long? DeviceId { get; set; }
    public string? DeviceCode { get; set; }
    public string? AlertContent { get; set; }
    /// <summary>1=提示 2=警告 3=严重</summary>
    public int? Level { get; set; }
    /// <summary>0=未处理 1=已处理</summary>
    public int? Status { get; set; }
    public string? Source { get; set; }
    public string? OrderNo { get; set; }
    public DateTime? CreateTime { get; set; }
    public DateTime? UpdateTime { get; set; }
}
